EXT_MD.md

```diff
@@ -0,0 +1,132 @@
+Domain-Specific Naming Conventions for Molecular Dynamics Simulation Codes
+=======================================================================
+
+openPMD extension name: `MD`
+
+VERSION: 0.0.3 (January 26th, 2020)
+
+Introduction
+------------
+
+This extension is specifically designed for the molecular dynamics simulation codes.
+
+The current version of this extension is suitable to allow the output of
+arbitrary simulation codes to be post-processed and compared with common
+tools and frameworks. Future versions will define a common set of required
+records and further attributes.
+
+The example data structure can be found [HERE](https://github.com/ejcjason/MDDomainExtension).
+
+Root Group
+---------------------------
+
+### Additional Attributes for the `Root` group
+
+The following additional attributes are defined in this extension.
+The individual requirement is given in `scope`.
+
+  - `forceField`
```

Comment:
for `forceField` and `forceFieldParameters`, do you want to define the syntax of the values further or keep values as a human-readable-only free text?
You can check out the [`SpeciesType`](https://github.com/openPMD/openPMD-standard/blob/upcoming-2.0.0/EXT_SpeciesType.md) extension for ideas how to define syntax and alternatives.

Reply:
The ideal case is that there could be a well-defined syntax to describe the forceField, and to be read by MD simulation code directly. But in practice, for different forceFields, there will be different parameters. Like the [LJ potential](https://lammps.sandia.gov/doc/pair_lj.html) and  [EAM potential](https://lammps.sandia.gov/doc/pair_eam.html), the parameter syntax are different. 

Since I cannot understand all the potentials used in MD simulation, maybe I can start with defining the syntax from some potentials I understand? If it's other potential, just make it a human-readable-only free text. 

Should we also ask the opinion of some other experts about this?

Reply:
Sure, feel free to reach out. Otherwise you can also just keep it undefined for now.

You can always add more attributes in openPMD and if you arrive at something that is worth standardizing for those attributes, then we can just specify them later.

Reply:
Actually, I don't particularly like this constraint in the name. ;)

Consider a case where one wishes to find the *ground state* of an atomic configuration. This is not a `forceField` but just a `method`. Would this make sense to change?

Reply:
What do you think @JunCEEE? Sounds reasonable to me, if storing the used method is the intent for this attribute :)